src/ai_marketplace_monitor/ai.py

```
@@ -366,3 +374,99 @@ class OllamaBackend(OpenAIBackend):
+                response = self.client.messages.create(
+                    model=self.config.model or self.default_model,
+                    max_tokens=1024,
+                    system="You are a helpful assistant that can confirm if a user's search criteria matches the item he is interested in.",
+                    messages=[
+                        {"role": "user", "content": prompt},
+                    ],
+                )
+                break
+            except KeyboardInterrupt:
+                raise
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)